sla_per_user_system_test.py

```diff
@@ -828,8 +829,7 @@ def _compare_workloads_c_s_metrics(self, workloads_queue: list) -> dict:
         comparison_axis = {"latency 95th percentile": 2.0,
                            "latency 99th percentile": 2.0,
                            "op rate": 2.0}
-        release = parse_version(self.db_cluster.nodes[0].scylla_version.replace("~", "-")).release
-        if release[0] > 2024 or (release[0] == 2024 and release[1] >= 3):
+        if ComparableScyllaVersion(self.db_cluster.nodes[0].scylla_version) >= '2024.2.0':
```

Comment:
For the moment the latest available version of the `2024.2.x` is `2024.2.0-rc2`.
So, to be able to make `rc` versions be included then need to use it in comparison as follows:
```
if ComparableScyllaVersion(self.db_cluster.nodes[0].scylla_version) >= '2024.2.0~rc2':
```

Or even dev version (if any `2024.2.x` has support for the needed feature):
```
if ComparableScyllaVersion(self.db_cluster.nodes[0].scylla_version) >= '2024.2.0~dev':
```

Reply:
got it, changed to 2024.2.0~rc3

Reply:
`rc3` is absent yet. So, are you sure about it? Maybe `rc2`?

Reply:
https://argus.scylladb.com/tests/scylla-cluster-tests/c4fbed30-dc6d-4eee-b901-d8cb7d3c6519 - has version rc3

Reply:
I used the following command:
```
$ ./docker/env/hydra.sh list-images -r eu-west-1 -v enterprise-2024.2
There is scylladb/hydra:v1.76-argus-result-screenshots-0.12.8 in local cache, using it.
Going to run './sct.py  list-images -r eu-west-1 -v enterprise-2024.2'...
...
+-----------------------------------------------------------------------------------------------------------------------------------------------------------+
|                                                     AWS Machine Images by Version in region eu-west-1                                                     |
+---------+----------------------------------+-----------------------+--------------------------+-----------------------------------------------------------+
| Backend | Name                             | ImageId               | CreationDate             | NameTag                                                   |
+---------+----------------------------------+-----------------------+--------------------------+-----------------------------------------------------------+
| AWS     | ScyllaDB Enterprise 2024.2.0-rc2 | ami-0555cb82c50d0d5f1 | 2024-09-09T11:02:54.000Z | scylla-enterprise-2024.2.0-rc2-x86_64-2024-09-05T08-22-30 |
| AWS     | ScyllaDB Enterprise 2024.2.0-rc1 | ami-03c86412a86a6cfb3 | 2024-07-24T12:03:38.000Z | scylla-enterprise-2024.2.0-rc0-x86_64-2024-07-16T15-45-02 |
| AWS     | ScyllaDB Enterprise 2024.2.0-rc0 | ami-0bca336fb539e1d78 | 2024-07-02T05:34:41.000Z | scylla-enterprise-2024.2.0-rc0-x86_64-2024-06-30T10-49-08 |
+---------+----------------------------------+-----------------------+--------------------------+-----------------------------------------------------------+
```

But ok, if it is not mistake, I don't have concerns.